docs/next.config.js

```diff
@@ -128,6 +128,7 @@ module.exports = {
     };
   },
   exportTrailingSlash: true,
```

Comment:
> if you want to force trailing slashes the recommendation is to just use trailingSlash: true

--https://github.com/vercel/next.js/discussions/15598#discussioncomment-43808

Sounds like we can remove this.

Reply:
I have tried without, it doesn't work correctly. We need to create a folder per route.

Reply:
Could you ask for clarification then? The statement makes it pretty clear that we only need one. It's not obvious how these interact and why we need both.

Reply:
`exportTrailingSlash` impacts the export output, it goes from `/autocomplete.html` to `/autocomplete/index.html`. On the other hand `trailingSlash` impacts how the links are generated.

Reply:
So while we can set `exportTrailingSlash: false` and `trailingSlash: true`, and have the navigation work, it leads to 301/308 redirection once you refresh the page.

Reply:
Actually, if we look at the documentation, it's what it's described. https://nextjs.org/docs/api-reference/next.config.js/exportPathMap#adding-a-trailing-slash, change the HTML output.

I think that https://nextjs.org/docs/api-reference/next.config.js/trailing-slash would work if we weren't hosted on a CDN.

Reply:
> On the other hand trailingSlash impacts how the links are generated. This is based on my observation, so far.

Why would this be different? If next already generates links based on a trailing slash why would it then not export them as such?

Reply:
> Why would this be different?

I think that in theory, `trailingSlash` is all we need. In practice, it's not enough. I think that it should lead to an issue on Next.js side. Let me do that.

Reply:
https://github.com/vercel/next.js/issues/15774